src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts

```diff
@@ -185,4 +185,15 @@ export class AddRobotComponent implements OnInit, OnDestroy {
         this.messageHandlerService.showSuccess(res);
       });
   }
+
+  radioChange(num): void {
+    if (num === 1) {
+      this.robot.access.isPullImage = true;
+      this.robot.access.isPushOrPullImage = false;
+    }
+    if ( num === 2) {
```

Comment:
why not use else if

Reply:
one 'if'  to one 'radio button', easy to read.

Reply:
These two IF are mutually exclusive,If num = 1    I don't need to execute if num = 2

Reply:
I agree with wangyang's comments, please use if ,else if instead. And please do not use magic number in the code. const definition will be preferred.

Reply:
modified .Thanks for your comments